src/Heading/index.js

```diff
@@ -31,6 +31,33 @@ const Anchor = ({ id }) => (
   />
 )
 
+const generateCss = (isHeading1, isHeading2) => {
+  if (isHeading1) {
+    return `& + p {
+      margin-top: var(--spectrum-global-dimension-static-size-300) !important;
+      font-size: var(--spectrum-global-dimension-static-size-225);
+    }`
+  } else if (isHeading2) {
+    return `&:nth-last-child(2) a:last-of-type {
```

Comment:
does this warrant a test? asking since the existing one for `Heading1` seems like it could be copy-pasta'd for the `nth-last-child(2)`

https://github.com/adobe/parliament-ui-components/blob/84bc98591b77341a92960c5445d47ac81841c487/src/Heading/test/Heading.test.js#L19-L27